lib/locale-de-DE.json

```diff
@@ -0,0 +1,46 @@
+{
+  "n": {
+    ".gitignore": "Ignorierdatei (.gitignore)",
+    "repository": "Projektordner (repository)",
+    "repositories": "Projektordner (repositories)",
+    "branch": "Zweig (branch)",
+    "branches": "Zweige (branches)",
+    "fork": "Kopie (fork)",
+    "forks": "Kopien (forks)",
+    "request": "Anfrage (request)",
+    "pull request": "Änderungsanfrage (pull request)",
+    "pull requests": "Änderungsanfragen (pull requests)",
+    "commit": "Änderungssatz (commit)",
+    "commits": "Änderungssätze (commits)",
+    "upstream": "ursprüngliches Projekt (upstream)",
+    "feature": "Funktionalität (feature)",
+    "status": "Status",
+    "diff": "Unterschiede (diff)",
+    "readme": "Hilfedatei (readme)",
```

Comment:
Here i would use the Term "Liesmich", which is also commonly used in German.
The file is in general not a Help-File, but a file to read with instructions, additional info, license terms or whatever. Therefore i would use the direct German translation instead of the name "Hilfedatei".